### ISSUE TITLE
what WAI does, ARIA, and such

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-intro-accessibility#19]
As I understand it, @brewerj  was looking for more information on what WAI does, for example, ARIA.

**Comment by shawna-slh** [comment]
Currently says:
> The W3C Web Accessibility Initiative ([WAI](https://www.w3.org/WAI/about.html)) develops guidelines, techniques, technical specifications, and supporting resources that describe accessibility solutions. These are considered the international standard for web accessibility; for example, WCAG 2.0 is also an ISO standard: ISO/IEC 40500."

What WAI does should be covered in About WAI (https://www.w3.org/WAI/about.html will be updated for redesign), which is linked from this page.

We're also adding ARIA to Essential Components doc [https://github.com/w3c/wai-components/issues/2 2], which is linked from this page.

Also will be some in the new page: "W3C Accessibility Standards Overview", which is linked from this page.

I'm thinking that given this is an Intro to Web Accessibility, not WAI, that there shouldn't be more about WAI in this resource.

If anyone disgrees, please re-open with specific suggestions.

Thanks.

**Comment by brewerj** [comment]
I saw this after I put a new comment into Github. Glad to hear you're adding ARIA to Essential Components. Please let me know when that's in place, hopefully also an update of the graphic. The gist of my comment isn't so much that I'm looking for a description of what WAI does (I'm not), but a representation that the scope of WAI work includes different types of technical resources, plus an approach in which one can get involved (even if just by commenting on resources one is using; e.g., there's participatory aspects throughout). And the W3C Accessibility Standards Overview isn't actually linked (it's 404) so it's hard to find what people would get when they chase that link. Please see my direct comments and suggestions here: 
https://github.com/w3c/wai-intro-accessibility/issues/20
